playwright-tests/pages/base.page.ts

```diff
@@ -24,32 +25,38 @@ export class BasePage {
     this.aboutUsDropdown = page.getByRole('button', { name: 'About Us' });
     this.menuitem = (itemTitle: string) =>
       page.getByRole('menuitem', { name: itemTitle });
+    this.menuItems = page.getByRole('listitem');
   }
-
   async navigateToPath(path: string) {
     await test.step(`Navigate to path: ${path}`, async () => {
       await this.page.goto(path);
       await this.page.waitForLoadState('networkidle');
-    });
+  });
   }
-
   async clickElement(elementLocator: Locator) {
     await test.step(`Click the "${elementLocator}" link`, async () => {
       await elementLocator.click();
     });
   }
-
   async verifyURL(expectedURL: string) {
     await test.step(`Verify URL is "${expectedURL}"`, async () => {
       await expect(this.page).toHaveURL(expectedURL);
     });
   }
-
   async verifyPageContainsText(expectedText: string) {
     await test.step(`Verify page contains text "${expectedText}"`, async () => {
       await expect(
         this.page.getByText(expectedText, { exact: true }),
       ).toBeVisible();
     });
   }
+   async openMentorshipDropdown() {
```

Comment:
I think it is missing the auto-format config, because does not look correct form this line.